doc/HOWTO_oracle.md

```diff
@@ -227,3 +230,98 @@ impdp \
+
+Add the following code to your `~/.profile`, `~/.zshrc` or `~/.bash_profile` depending on your terminal's platform.
+
+```
+if [ "$(uname -m)" = "arm64" ]; then
+  echo "arm64"
+  export PATH="/opt/homebrew/bin:${PATH}"
+  [ -f /opt/homebrew/bin/brew ] && eval "$(/opt/homebrew/bin/brew shellenv)"
+else
+  echo "$(uname -m)"
+  # Postgres Caveats
+  export PATH="/usr/local/opt/libpq/bin:$PATH"
+  export LDFLAGS="-L/usr/local/opt/libpq/lib"
+  export CPPFLAGS="-I/usr/local/opt/libpq/include"
+  export PKG_CONFIG_PATH="/usr/local/opt/libpq/lib/pkgconfig"
+  # Homebrew
+  export PATH="/usr/local/bin:${PATH}"
+
+  [ -f /usr/local/bin/brew ] && eval "$(/usr/local/bin/brew shellenv)"
+fi
+```
+
+After saving this, every time you open a new terminal it should print the current architecture.
+
+2. Install Homebrew in the native terminal
+* This step you'll probably have it done already, if so just skip it
+* Run `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
+  -  Make sure the root of the install is **/opt/homebrew/**
+
+
+3. Install Homebrew in the x86 terminal
+
+* First we need to change our terminal's architecture
+  - Run this on your terminal `arch -x86_64 /bin/zsh`
+  - The output should print "x86_64"
+  - You can check your actual architecture by running `arch`, if the output is "i386" you're good to go
+
+* Now you can install Homebrew normaly
+  - Run `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
+  - Make sure the root of the install is **/usr/local/**
+
+4. Install ruby the ruby version of your current architecture
+
+* If you had the arm64 ruby version already installed it won't allow you to install Ruby x86 with the plain rvm install command as it will detect that version is already installed.
+
+* We need to call it a different name and make sure it uses the desired architecture.
+  - Run `rvm install ruby-3.3.0 --name ruby-3.3.0-x86_64 --with-architecture=x86_64`
+
+* Once installed make sure to use this ruby instance
+  - Run `rvm use ruby-3.3.0-x86_64`
+
+5. If ruby-oci8 is not founding Oracle instant client installed you can install it manually and specifying the route where Oracle instant client is located.
+As the Oracle instant client was installing before setting it will be located at the arm64 Homebrew path.
+
+  - Run `export DYLD_LIBRARY_PATH=/opt/homebrew/Cellar/instantclient-basic/19.8.0.0.0dbru/lib`
+  - Run `gem install ruby-oci8 -- --with-instant-client-dir=/opt/homebrew/Cellar/instantclient-basic/19.8.0.0.0dbru`
+
+This are example path, please make sure to type your specific path.
+
+### Oracle instant client folder structure
+
+It can happen that you have Oracle instant client correctly installed but ruby-oci8 can't find specific files on the instantclient-basic folder. This is happens as Oracle instant client consists of different packages and they may be stored in different folders.
```

Comment:
```suggestion
It can happen that you have Oracle instant client correctly installed but ruby-oci8 can't find specific files on the instantclient-basic folder. This happens as Oracle instant client consists of different packages and they may be stored in different folders.
```